nodl_python/nodl/_util.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it under the terms of the
+# GNU Limited General Public License version 3, as published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranties of MERCHANTABILITY, SATISFACTORY QUALITY, or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Limited General Public License for more details.
+#
+# You should have received a copy of the GNU Limited General Public License along with
+# this program. If not, see <http://www.gnu.org/licenses/>.
+
+from distutils.util import strtobool
```

Comment:
I'd prefer an `import distutils.util` and then using `distutils.util.strtobool`. Keeps it clear where things are coming from (compare to abuse of `using namespace` in C++). I realize this module is short, but something to keep in mind.

Reply:
Updated